django/core/management/commands/test.py

```diff
@@ -48,6 +48,12 @@ def add_arguments(self, parser):
             help='Overrides the default address where the live server (used '
                  'with LiveServerTestCase) is expected to run from. The '
                  'default value is localhost:8081-8179.'),
+        parser.add_argument('--tag', action='append', dest='tags',
+            help='Tells Django to only run tests having specified tags. '
```

Comment:
I think "Tells Django to" is useless here. How about just "Run only tests with the specified tag. You can use this option multiple times."